### ISSUE TITLE
Navigation Cube not working

### ISSUE THREAD

**Issue by synsuiter** [SynDaverCO/symple-slicer#181]
**Describe the bug**
The top right navigation cube no longer responds when used to view faces of our build volume. 

**Details**
 - SympleSlicer Version: V1.2.0
 - Browser : Chrome

**To Reproduce**
1. Open SS
2. Use the Navigation Cube

**Expected behavior**
It should respond to the selection

**Screenshots**
![nav-cube](https://user-images.githubusercontent.com/65256117/194366003-ee0d29b1-448c-425c-8809-61b136546f31.PNG)

**Comment by synsuiter** [comment]
When I confirmed each section it was all working fine until I chose "Back". Once I chose that one the front and back sections went blank on the cube. When I moved the view around I saw the other sides were still visible.

During this, all options still seemed to work, the issue seems to be visual now not functional.

![hollow-nav-cube](https://user-images.githubusercontent.com/65256117/195882187-0054c032-d0bb-4235-8d49-faa400323209.PNG)
